src/docs/sphinx/advancedExamples/validationStudies/mccWellbore/Example.rst

```diff
@@ -0,0 +1,235 @@
+.. _ExampleMCCWellbore:
+
+
+####################################################
+Modified Cam-Clay Model for Wellbore Problems
+####################################################
+
+
+**Context**
+
+In this benchmark example, the Modified Cam-Clay model (see :ref:`ModifiedCamClay`) is applied to solve for elastoplastic deformation within the vicinity of a vertical wellbore. For the presented example, an analytical solution is employed to verify the accuracy of the numerical results. The resulting model can be used as a base for more complex analysis (e.g., wellbore drilling, fluid injection and storage scenarios). 
+
+
+**Input file**
+
+Everything required is contained within two GEOSX input files located at:
+
+.. code-block:: console
+
+  src/coreComponents/physicsSolvers/solidMechanics/benchmarks/PlasticWell_Base.xml
+  src/coreComponents/physicsSolvers/solidMechanics/benchmarks/ModifiedCamClay_Verification.xml
+
+
+------------------------------------------------------------------
+Description of the case
+------------------------------------------------------------------
+
+We simulate a drained wellbore problem subjected to isotropic horizontal stress (:math:`\sigma_h`) and vertical stress (:math:`\sigma_v`), as shown below. By increasing the wellbore supporting pressure (:math:`P_w`), the wellbore expands, and the formation rock experiences elastoplastic deformation. A plastic zone develops in the near wellbore region.
+
+
+.. _problemSketchFig:
+.. figure:: MCC_wellSketch.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Sketch of the wellbore problem 
+
+
+To simulate this phenomenon, the Modified Cam-Clay model is used in this example. Displacement and stress fields around the wellbore are numerically calculated. These numerical predictions are then compared with the corresponding analytical solutions `(Chen and Abousleiman, 2013)  <https://www.icevirtuallibrary.com/doi/10.1680/geot.11.P.088>`__ from the literature. 
```

Comment:
Please provide a resume of these analytical results. Also, please add a Python script for plotting them instead of inserting a hard figure.

Reply:
Reference of the analytical solutions has already been given and the corresponding parameters have been summarized in the table. The python script will be included.